pom.xml

```diff
@@ -81,6 +81,16 @@
       <artifactId>jnr-posix-api</artifactId>
       <version>3.1.7-1</version>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>commons-text-api</artifactId>
+      <version>1.10.0-36.vc008c8fcda_7b_</version>
```

Comment:
Version should not be provided. It's part of bom

Reply:
Refer to [this](https://github.com/jenkinsci/datadog-plugin/pull/404#discussion_r1555711413)